_data/domains.yml

```diff
@@ -63,6 +63,15 @@ websites:
       twitter: CrazyDomains
       facebook: crazydomains
 
+    - name: CSC
```

Comment:
Is it CSC or CSC Global?

Reply:
Publically we go by CSC. Unfortunately, the csc domain name was owned by the other CSC (computer services corporation) which was bought out by DXC Technology. Long story short - we are CSC, but our website is cscglobal.com.